include/zephyr/net/socket_service.h

```diff
@@ -81,8 +81,6 @@ struct net_socket_service_desc {
 #define __z_net_socket_svc_get_idx(_svc_id) __z_net_socket_service_idx_##_svc_id
```

Comment:
The commit subject `net: sockets: socket_service: ...` has redundant information, could be simplified to
`net: socket: service: Optimize code`